### ISSUE TITLE
Crash in [PPAuthViewController findLoginFrom1Password:]

### ISSUE THREAD

**Issue by JordanB** [paypal/PayPal-iOS-SDK#427]
### Required Information

- Mode: Live
- PayPal iOS SDK Version: 2.13.1
- iOS Version and Device: iOS 9 at 96% and iOS 8 at 4%
- PayPal-Debug-ID(s) (from any logs): Not sure what that is

### Issue Description

```
Thread 0 Crashed:
0   CoreFoundation                 0x000000018371eebc CFStringGetLength + 20
1   CoreFoundation                 0x000000018373def4 CFCharacterSetCreateWithCharactersInString + 32
2   CoreFoundation                 0x000000018373def4 CFCharacterSetCreateWithCharactersInString + 32
3   Foundation                     0x0000000184125144 -[NSCharacterSet characterSetWithCharactersInString:] + 20
4   UberClient                     0x00000001009c2fb0 -[PPAuthViewController findLoginFrom1Password:] + 172
5   UberClient                     0x00000001009b5c18 -[PPZebraOnePasswordExtension findLoginForURLString:forViewController:sender:completion:] + 60
6   UberClient                     0x00000001009b7a40 -[PPZebraOnePasswordExtension processExtensionItem:completion:] + 168
7   libextension.dylib             0x00000001840e6edc -[NSItemProvider _loadItemOfClass:withLoadHandler:options:coerceForCoding:completionHandler:] + 260
8   libextension.dylib             0x00000001840e6dcc -[NSItemProvider _loadItemOfClass:withLoadHandler:options:coerceForCoding:completionHandler:] + 80
9   libextension.dylib             0x00000001840e6a88 -[NSItemProvider _loadItemOfClass:forTypeIdentifier:options:coerceForCoding:completionHandler:] + 312
10  libdispatch.dylib              0x000000018328d630 _dispatch_call_block_and_release + 24
11  libdispatch.dylib              0x000000018328d5f0 _dispatch_client_callout + 16
12  libdispatch.dylib              0x0000000183292cf8 _dispatch_main_queue_callback_4CF + 1844
13  CoreFoundation                 0x00000001837f0bb0 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 12
14  CoreFoundation                 0x00000001837eea18 __CFRunLoopRun + 1628
15  CoreFoundation                 0x000000018371d680 CFRunLoopRunSpecific + 384
16  GraphicsServices               0x0000000184c2c088 GSEventRunModal + 180
17  UIKit                          0x0000000188594d90 UIApplicationMain + 204
18  UberClient                     0x00000001008b0208 main + 260
19  libdyld.dylib                  0x00000001832be8b8 start + 4
```

**Comment by bluk** [comment]
Do you have 1Password installed and is there a valid entry for 1Password (e.g. is the username or password empty)?

**Comment by JordanB** [comment]
I don't have this information because I don't know how the users manage to produce this crash. All I know is that we had 25 crashes for 22 users, which is a very low impact for Uber. Still if there is a way for you guys to see where this could potentially crash and make sure it never does, that would be great. Let me know if I can help.

**Comment by bluk** [comment]
This issue should be fixed in [release 2.14.3](https://github.com/paypal/PayPal-iOS-SDK/releases/tag/2.14.3). If it is still an issue, please leave a comment. Thanks for your patience.
